conformance.adoc

```diff
@@ -561,29 +563,25 @@ particular value of the **`axis`** attribute.
 * The **`grid_mapping`** and **`coordinates`** attributes can be carried by the
 geometry container variable provided they are also carried by the data variables
 associated with the container.
-* If a **`coordinates`** attribute is carried by the geometry container variable, then 
-those coordinate variables which correspond to node coordinate variables must also have 
-the **`axis`** attribute.
+* If a **`coordinates`** attribute is carried by the geometry container variable or its parent data
+variable, then those coordinate variables which correspond to node coordinate variables must have a
+ **`bounds`** attribute that names the corresponding node coordinate.
 * The geometry node coordinate variables must all have the same single dimension,
 which is the total number of nodes in all the geometries.
 * The nodes must be stored consecutively for each geometry and in the order of the
 geometries, and within each multipart geometry the nodes must be stored
 consecutively for each part and in the order of the parts.
 * Nodes for polygon exterior rings must be put in anticlockwise order (viewed from above)
 and polygon interior rings in clockwise order.
```

Comment:
The above two rules concerning the order of nodes are not checkable, so shouldn't they be omitted from the conformance requirements? These points are already made in chapter 7. ​

Reply:
> * The nodes must be stored consecutively for each geometry...

That one is uncheckable.  I agree with omitting it.

> * Nodes for polygon exterior rings must be put in anticlockwise order...

This one can be checked.  [Here's a checker in Python](https://github.com/twhiteaker/netCDF-CF-simple-geometry/blob/master/src/python/ncgeom/part.py#L98).

Reply:
That's good for me. Thanks for the link to your geometry code.

Reply:
I removed the uncheckable rule.  You can find the latest version here:
https://github.com/cf-convention/Conformance/pull/6/files